addons/website/static/tests/tours/website_no_dirty_page.js

```diff
@@ -24,6 +24,9 @@ tour.register('website_no_dirty_page', {
         content: "Confirm we are in edit mode",
         trigger: 'body.editor_has_snippets',
         run: () => null,
+    }, {
+        content: "Click on default paragraph",
+        trigger: '.s_text_image p.o_default_snippet_text',
```

Comment:
_Nitpicking/Can ignore:_
Maybe you want to add an extra step in the step below which ensures the `o_default_snippet_text` is still there?
Right now, it indirectly test it but the editor could later consider those class removal as not dirty and let you discard while the class was still removed? (would not be that bad and surely overthinking here?)

Reply:
Could do that, although not the point of this test. But I have to create a task to review the default text behavior anyway so I'll do this:
- Add the extra trigger, with a TODO comment "to move in other text once feature reviewed"
- Mention that in the task I'll create for master, we can make a proper test about this at the time.